angular.json

```diff
@@ -93,7 +98,47 @@
           }
         }
       }
+    },
+    "jazz-lib": {
```

Comment:
Maybe go with "design-system-lib" instead of "jazz-lib" to avoid using the marketing name.